Master is failing to build

Getting this after I merged one of @justinsb's PRs.

```
I0220 17:39:25.932952   12799 target.go:170] Cloudformation output is in /tmp/test797552248/out
--- FAIL: TestMinimalCloudformation (0.96s)
	integration_test.go:303: diff:
		...
		              {
		                "Action": [
		+                 "route53:ChangeResourceRecordSets",
		+                 "route53:ListResourceRecordSets",
		+                 "route53:GetHostedZone"
		+               ],
		+               "Effect": "Allow",
		+               "Resource": [
		+                 "arn:aws:route53:::hostedzone/Z1AFAKE1ZON3YO"
		+               ]
		+             },
		+             {
		+               "Action": [
		+                 "route53:GetChange"
		+               ],
		+               "Effect": "Allow",
		+               "Resource": [
		+                 "arn:aws:route53:::change/*"
		+               ]
		+             },
		+             {
		+               "Action": [
		+                 "route53:ListHostedZones"
		-                 "route53:*"
		                ],
		                "Effect": "Allow",
		...
		              {
		                "Action": [
		+                 "route53:ChangeResourceRecordSets",
		+                 "route53:ListResourceRecordSets",
		+                 "route53:GetHostedZone"
		+               ],
		+               "Effect": "Allow",
		+               "Resource": [
		+                 "arn:aws:route53:::hostedzone/Z1AFAKE1ZON3YO"
		+               ]
		+             },
		+             {
		+               "Action": [
		+                 "route53:GetChange"
		+               ],
		+               "Effect": "Allow",
		+               "Resource": [
		+                 "arn:aws:route53:::change/*"
		+               ]
		+             },
		+             {
		+               "Action": [
		+                 "route53:ListHostedZones"
		-                 "route53:*"
		                ],
		                "Effect": "Allow",
		...
		
	integration_test.go:305: cloudformation output differed from expected
I0220 17:39:25.940136   12799 zones.go:73] ListHostedZonesPages {
}
```

--- a/tests/integration/minimal/cloudformation.json
+++ b/tests/integration/minimal/cloudformation.json
@@ -538,7 +538,27 @@
             },
             {
               "Action": [
-                "route53:*"
+                "route53:ChangeResourceRecordSets",
+                "route53:ListResourceRecordSets",
+                "route53:GetHostedZone"
+              ],
+              "Effect": "Allow",
+              "Resource": [
+                "arn:aws:route53:::hostedzone/Z1AFAKE1ZON3YO"
+              ]
+            },
+            {
+              "Action": [
+                "route53:GetChange"
+              ],
+              "Effect": "Allow",
+              "Resource": [
+                "arn:aws:route53:::change/*"
+              ]
+            },
+            {
+              "Action": [
+                "route53:ListHostedZones"
               ],
               "Effect": "Allow",
               "Resource": [
@@ -593,7 +613,27 @@
             },
             {
               "Action": [
-                "route53:*"
+                "route53:ChangeResourceRecordSets",
+                "route53:ListResourceRecordSets",
+                "route53:GetHostedZone"
+              ],
+              "Effect": "Allow",
+              "Resource": [
+                "arn:aws:route53:::hostedzone/Z1AFAKE1ZON3YO"
+              ]
+            },
+            {
+              "Action": [
+                "route53:GetChange"
+              ],
+              "Effect": "Allow",
+              "Resource": [
+                "arn:aws:route53:::change/*"
+              ]
+            },
+            {
+              "Action": [
+                "route53:ListHostedZones"
               ],
               "Effect": "Allow",
               "Resource": [

